org.rdkit.knime.bin.linux.x86/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: RDKit binaries for 32bit Linux
 Bundle-SymbolicName: org.rdkit.knime.bin.linux.x86;singleton:=true
-Bundle-Version: 3.4.0.qualifier
+Bundle-Version: 3.5.0.qualifier
 Bundle-Vendor: NIBR
 Fragment-Host: org.rdkit.knime.types;bundle-version="3.4.0"
```

Comment:
I would also change the dependency version for org.rdkit.knime.types to 3.5.0 - otherwise we may see inconsistencies in strange installs